storage/replica_range_lease.go

```diff
@@ -124,6 +125,9 @@ func (p *pendingLeaseRequest) InitOrJoinRequest(
 		ba.Timestamp = replica.store.Clock().Now()
 		ba.RangeID = replica.RangeID
 		ba.Add(leaseReq)
+		if log.V(2) {
+			log.Infof(ctx, "%s: sending lease request %v", replica, leaseReq)
```

Comment:
It's not equivalent.. `VEventf` would cause it to show up as an event in any trace, the current code does not. I assume we want to keep as-is (the event sounds verbose).